source/deployment/ldap-group-sync.rst

```diff
@@ -0,0 +1,146 @@
+
+If you see a “Linked Failed” message, either click on the message or check the box before the group name to expose the inline link message, allowing you to try the link again.
+
+.. image:: ../images/LinkFailed.png
+
+Configure the group
+-------------------
+
+AD/LDAP groups that have been linked to Mattermost groups are able to be configured.  To configure the group, select “Configure”.  This will open up the Group Configuration page.  
+
+The Group Configuration page displays the group profile which includes the group name.  This name is automatically mapped from the AD/LDAP group common name attribute and is read-only.  
+
+Add default teams or channels for the group
+--------------------------------------------
+To add the default Team or Channels that you want the group members to default in, select either “Add Team” or “Add Channel” from the blue “Add Team or Channel” button.  You may select multiple teams and channels.  
+
+.. image:: ../images/Add_Team_Or_Channel.png
+
+Channels are nested below the Team they belong to in the team and channel list.  
+
+Teams that are open for anyone to join are indicated by:
+ 
+.. image:: ../images/open_team.png  
+   
+Teams that are not open for anyone to join are indicated by:
+ 
+.. image:: ../images/private_team.png 
+ 
+Public channels are indicated by: 
+ 
+.. image:: ../images/public_channel.png
+
+Private channels are indicated by:
+ 
+.. image:: ../images/private_channel.png  
+
+Teams added include default channels set in the `ExperimentalDefaultChannels config setting <https://docs.mattermost.com/administration/config-settings.html?highlight=configuration%20settings#default-channels-experimental>`_ (Town Square and Off-Topic). Adding a channel without setting the team adds the implied team to the listing below, but not to the group specifically.  Teams are listed in parentheses after the channel name in the channel selector.
+
+Teams and channels membership synchronization
+----------------------------------------------
+
+Default teams and channels will be added to the group member's user interface when they login for the first time after the group has been configured with the specified teams and channels. It may take a few seconds to load all team and channel memberships for a user depending on the number of teams and channels the group is defaulted to. In our testing, it took 6s for an organization with 200,000 users and 30,000 linked groups. This process is independent of the AD/LDAP synchronization process to support future functionality of groups.
```

Comment:
"Default teams and channels will be added to the group member's user interface when they login for the first time after the group has been configured with the specified teams and channels."

1. To confirm, is this only after the first time they log in? (Is it that they are added to the Mattermost group on next synchronization, but only added to new channels on next login?) 
2. just a note, login is two words when it's a verb, one when it's a noun 
3. Can we write out "seconds", and I think our docs guideline also asks that we write out "six" - https://docs.mattermost.com/process/documentation-guidelines.html#numbers 

"This process is independent of the AD/LDAP synchronization process to support future functionality of groups."
1. I'm not sure what this means

Reply:
Rewrote some and re-organized.  Removed the "process is independent of the ad/ldap synchronization" after some re-clarification from Martin (this was not where we ended up in Phase 1, but likely will be for later phases).